### PR TITLE
[HLT] [LLVM14] Apply clang-tidy and clang-format changes

### PR DESCRIPTION
**Pull request by smuzaffar**
code-checks/format changes needed due to llvm 14.0.6 integration.

### CODE REVIEW

File: HLTrigger/special/plugins/HLTHcalSimpleRecHitFilter.cc [hunk 1]
```diff
@@ -71,15 +71,15 @@ HLTHcalSimpleRecHitFilter::HLTHcalSimpleRecHitFilter(const edm::ParameterSet& iC
...
-      edm::LogError("cfg error") << "masked list of channels missing from HLT menu. Migration from vector of ints to "
-                                    "vector of uints needed for this release";
-      cms::Exception("Invalid/obsolete masked list of channels");
+    //worry about possible user menus with the old interface
+    if (iConfig.existsAs<std::vector<int> >("maskedChannels")) {
+      std::vector<int> tVec = iConfig.getParameter<std::vector<int> >("maskedChannels");
+      if (!tVec.empty()) {
+        edm::LogError("cfg error") << "masked list of channels missing from HLT menu. Migration from vector of ints to "
+                                      "vector of uints needed for this release";
+        cms::Exception("Invalid/obsolete masked list of channels");
```
Comment (missirol) [inline HLTrigger/special/plugins/HLTHcalSimpleRecHitFilter.cc:80]: For the record, this is an old plugin that does not run in any recent HLT menu. (and this exception is not even `throw`n..)

**Comment by missirol** [comment]
please test

**Comment by missirol** [comment]
+hlt

File: HLTrigger/special/plugins/HLTHcalSimpleRecHitFilter.cc (hunk 1 above)
Comment (perrotta) [reply HLTrigger/special/plugins/HLTHcalSimpleRecHitFilter.cc:80]: So, maybe it is time to remove it from the release, and avoid having to maintain it...

**Comment by perrotta** [comment]
+1

**Pull request merged**